hw6/cmd.sql

```diff
@@ -0,0 +1,20 @@
+USE baseball;
+
+create or replace temporary table rolling_average_intermediate
+(SELECT a.batter, b.local_date, A.hit,A.atBat
+FROM batter_counts as A
+LEFT JOIN GAME as B
+ON A.game_id = B.game_id);
+CREATE INDEX batter ON rolling_average_intermediate(batter);
```

Comment:
Nice update on  your intermediate table, by adding indexs. This is going to make the join a lot faster